docs/index.rst

```diff
@@ -1,34 +1,96 @@
-Flocker Documentation
-=====================
+.. raw:: html
 
-Welcome to the Flocker docs! If you're just getting started with Flocker, it might be worth reading our :ref:`introduction`, or trying our simple tutorial on the `Getting Started with Flocker`_ page.
+    <style>
+        .toctree-wrapper { display:none; }
+    </style>
 
-In the documentation listed below, you'll be able to find instructions on how to install, configure, and use Flocker.
-You will also find instructions on how to install and use :ref:`docker-plugin`.
+==================
+Installing Flocker
+==================
 
-You'll be able to find concept and reference material, and our :ref:`Labs section<labs-projects>` contains all the information you need about our experimental projects.
+.. _supported-orchestration-frameworks:
 
-Flocker is an open-source project, and in our :ref:`Getting Involved section<getting-involved>`, there is information about how you can collaborate with ClusterHQ.
+Flocker allows you to launch and move stateful containers by integrating with a cluster manager of your choice.
```

Comment:
:question: I wasn't familiar with the term "cluster manager". Is it a standard term? Elsewhere we talk about schedulers, orchestration frameworks.